### PR TITLE
some minor modernizations, localization, spit and polish from a fan.  🍻 

### PR DESCRIPTION
**Pull request by zer0Kerbal**
## version = 1.11.3.11 - Spit and Polish?
* zer0Kerbal played with one of his favorite mods
* added masTerTourch/Localization/ #1 
* added localization (en-us.cfg) #1 
* adjusted phrasing a smudge
* - electostatic -> electrostatic
* - incomming -> incoming
* - phosporic -> phosphoric
* - munar -> Munar
* ScienceDef added: SunInSpaceHigh, SunInSpaceLow
* moved art into masTerTorch/Assets/
* updated texture pointers in model (original, png, dds included)
* updated textures (both 512 and 1024 version of .png and .dds included)
* updated part.cfg:
* - replaced mesh = with MODEL{}
* - added explosionPotential
* created Changelog.cfg [KERBALCHANGELOG] (.this)
* added additional fields in .version (might need to tweak urls)
* added shields to Readme.md(Bob likes graphics)

### CODE REVIEW

**Comment by mwerle** [comment]
Thanks for this! Will review and merge when I get back (diving with hammerheads for a ling weekend .)

On February 22, 2020 7:17:54 AM GMT+09:00, zer0Kerbal <notifications@github.com> wrote:
>## version = 1.11.3.11 - Spit and Polish?
>* zer0Kerbal played with one of his favorite mods
>* added masTerTourch/Localization/
>* added localization (en-us.cfg)
>* adjusted phrasing a smudge
>* - electostatic -&gt; electrostatic
>* - incomming -&gt; incoming
>* - phosporic -&gt; phosphoric
>* - munar -&gt; Munar
>* ScienceDef added: SunInSpaceHigh, SunInSpaceLow
>* moved art into masTerTorch/Assets/
>* updated texture pointers in model (original, png, dds included)
>* updated textures (both 512 and 1024 version of .png and .dds
>included)
>* updated part.cfg:
>* - replaced mesh = with MODEL{}
>* - added explosionPotential
>* created Changelog.cfg [KERBALCHANGELOG] (.this)
>* added additional fields in .version (might need to tweak urls)
>* added shields to Readme.md(Bob likes graphics)
>You can view, comment on, or merge this pull request online at:
>
>  https://github.com/mwerle/kdex/pull/2
>
>-- Commit Summary --
>
>  * 1.11.3.11
>  * Update README.md
>  * Update README.md
>  * Update README.md
>  * Update README.md
>
>-- File Changes --
>
>    A Changelog.cfg (34)
>    R GameData/masTerTorch/Assets/model.mu (0)
>    A GameData/masTerTorch/Assets/model000.dds (0)
>    A GameData/masTerTorch/Changelog.cfg (34)
>    A GameData/masTerTorch/Localization/en-us.cfg (104)
>    D GameData/masTerTorch/Parts/kdex/model000.mbm (0)
>    M GameData/masTerTorch/Parts/kdex/part.cfg (39)
>    M GameData/masTerTorch/kdex.version (5)
>    M README.md (13)
>    A kdex.version (38)
>    A model.zip (0)
>    A model000-1024x1024.zip (0)
>    A model000-512x512.zip (0)
>
>-- Patch Links --
>
>https://github.com/mwerle/kdex/pull/2.patch
>https://github.com/mwerle/kdex/pull/2.diff
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/mwerle/kdex/pull/2

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by zer0Kerbal** [comment]
> 
> Thanks for this! Will review and merge when I get back (diving with hammerheads for a ling weekend .)
> […](#)
> On February 22, 2020 7:17:54 AM GMT+09:00, zer0Kerbal ***@***.***> wrote: ## version = 1.11.3.11 - Spit and Polish? * zer0Kerbal played with one of his favorite mods * added masTerTourch/Localization/ * added localization (en-us.cfg) * adjusted phrasing a smudge * - electostatic -&gt; electrostatic * - incomming -&gt; incoming * - phosporic -&gt; phosphoric * - munar -&gt; Munar * ScienceDef added: SunInSpaceHigh, SunInSpaceLow * moved art into masTerTorch/Assets/ * updated texture pointers in model (original, png, dds included) * updated textures (both 512 and 1024 version of .png and .dds included) * updated part.cfg: * - replaced mesh = with MODEL{} * - added explosionPotential * created Changelog.cfg [KERBALCHANGELOG] (.this) * added additional fields in .version (might need to tweak urls) * added shields to Readme.md(Bob likes graphics) You can view, comment on, or merge this pull request online at: #2 -- Commit Summary -- * 1.11.3.11 * Update README.md * Update README.md * Update README.md * Update README.md -- File Changes -- A Changelog.cfg (34) R GameData/masTerTorch/Assets/model.mu (0) A GameData/masTerTorch/Assets/model000.dds (0) A GameData/masTerTorch/Changelog.cfg (34) A GameData/masTerTorch/Localization/en-us.cfg (104) D GameData/masTerTorch/Parts/kdex/model000.mbm (0) M GameData/masTerTorch/Parts/kdex/part.cfg (39) M GameData/masTerTorch/kdex.version (5) M README.md (13) A kdex.version (38) A model.zip (0) A model000-1024x1024.zip (0) A model000-512x512.zip (0) -- Patch Links -- https://github.com/mwerle/kdex/pull/2.patch https://github.com/mwerle/kdex/pull/2.diff -- You are receiving this because you are subscribed to this thread. Reply to this email directly or view it on GitHub: #2
> -- Sent from my Android device with K-9 Mail. Please excuse my brevity.

Cebu? Love diving - just haven't done it for a spell.

File: GameData/masTerTorch/Localization/en-us.cfg [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	#KDex_endEventGUIName = KDex: Lid Close
+	#KDex_actionGUIName = KDex: Toggle Lid
+	
+	#KDex_experimentActionName = KDex: Run Dust Analysis
+	#KDex_resetActionName = KDex: Discard Data
+	#KDex_reviewActionName = KDex: Review Data
+	#KDex_collectActionName = KDex: Take Data
+
+	// Kerbal Dust Experiment: Dust Analysis ScienceDefs (can be moved to another file)
+	#KDex_title = Dust Analysis
```
Comment (mwerle) [inline GameData/masTerTorch/Localization/en-us.cfg:21]: Same as #KDEX_title above (apart from capitalisation).  Could cause confusion..

File: GameData/masTerTorch/Localization/en-us.cfg [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+{
+	en-us
+	{
+		// KDex
+	#KDEX_title = Kerbal Dust Experiment (KDex)
+	#KDEX_manufacturer = masTerTorch R&D Center
+	#KDEX_description = Inside this tiny coffee can is a highly sensitive sensor which can measure the density of dust particles colliding with the back of the can. Now we can extract science out of collisions!
+	#KDEX_tags = experiment dust collision research rerun re-run reus re-us redo re-do reset re-set science sensor kdex
+
+	#KDex_startEventGUIName = KDex: Lid Open
```
Comment (mwerle) [inline GameData/masTerTorch/Localization/en-us.cfg:11]: Why the change from KDEX_ to KDex_ as the prefix?

And: is the localisation system case-sensitive in either case?

File: GameData/masTerTorch/Parts/kdex/part.cfg [hunk 3]
```diff
@@ -15,10 +19,10 @@ PART
 	cost = 1200
 	category = Science
 	subcategory = 0
-	title = Kerbal Dust Experiment
-	manufacturer = masTerTorch R&D Center
-	description = Inside this tiny coffee can is a highly sensitive sensor which can measure the density of dust particles colliding with the back of the can. Now we can extract science out of collisions!
-	tags = experiment dust collision research rerun re-run reus re-us redo re-do reset re-set science sensor
+	title = #KDEX_title // Kerbal Dust Experiment
+	manufacturer = #KDEX_manufacturer // masTerTorch R&D Center
+	description #KDEX_description // Inside this tiny coffee can is a highly sensitive sensor which can measure the density of dust particles colliding with the back of the can. Now we can extract science out of collisions!
```
Comment (mwerle) [inline GameData/masTerTorch/Parts/kdex/part.cfg:24]: missing "="

File: GameData/masTerTorch/kdex.version [hunk 4]
```diff
@@ -12,7 +15,7 @@
         "MAJOR":1,
```
Comment (mwerle) [inline GameData/masTerTorch/kdex.version:15]: Should possibly increase the major version here as this breaks compatibility with past versions of KSP.
Or at least increase minor version to V1.12

File: README.md [hunk 5]
```diff
@@ -25,15 +26,17 @@ Lots of results for the new experiment.
...
 * Forum threads
   + [Micha's Release Thread](http://forum.kerbalspaceprogram.com/index.php?showtopic=166619)
   + [masTerTorch's original Thread](http://forum.kerbalspaceprogram.com/index.php?showtopic=50039)
 * Download Links
-  + [GitHub](http://github.com/mwerle/kdex/releases)
+  + [GitHub](http://github.com/mwerle/kdex/releases/latest)
   + [SpaceDock](http://spacedock.info/mod/1551)
   + [CurseForge](https://www.curseforge.com/kerbal/ksp-mods/kdex-continued)
+  
+<!-- zed'K was here. ;p -->
```
Comment (mwerle) [inline README.md:42]: Best to add a "Contributors" section and put your name up in lights ;)

Comment (mwerle) [reply README.md:42]: And/or add a Contributors.md file and link it here.

File: Changelog.cfg [hunk 6]
```diff
@@ -0,0 +1,34 @@
+
+KERBALCHANGELOG //Required to have this name
+{
+	showChangelog = True //To show the changelog, this must be set to True
+	modName = KDex //Add your mod's name here
+	VERSION //Declares a version node
+	{
+		version = 1.11.3.11 //Version number, numbers only with no spaces!
+		change = zer0Kerbal played with one of his favorite mods
+		change = added masTerTourch/Localization/
```
Comment (mwerle) [inline Changelog.cfg:10]: masTerTorch (sp)

File: Changelog.cfg [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+		change = added localization (en-us.cfg)
+		change = adjusted phrasing a smudge
+		change = - electostatic -> electrostatic
+		change = - incomming -> incoming
+		change = - phosporic -> phosphoric
+		change = - munar -> Munar
```
Comment (mwerle) [inline Changelog.cfg:16]: ....
change = Added localisation and fixed some typos.
change = Updated model textures and tweaked part configuration.
change = Modernised mod layout and added additional metadata (Changelog)

(Less is more?)

File: GameData/masTerTorch/Localization/en-us.cfg [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+	#KDex_BopSrfLanded = Too much dust in the sensor.
+
+	#KDex_PolInSpaceLow = Dusting this one would be a good start.
+	#KDex_PolSrfLanded = Weak impacts. Probably phosphoric or sulfuric origin.
+
+	#KDex_EelooInSpaceLow = Dust particles are few and far between. 
+	#KDex_EelooSrfLanded = The Dust consists of CO, silicon carbide, amorphous silicate and lots of ice.
+	
+	#KDEx_SunInSpaceLow = Dust particles are popping like popped corn; perhaps from being too close to Kerbal?!?!?
+	#KDEx_SunInSpaceHigh = You feel kind of small right now... You hope you know where you're going. Dust particles are popping like popped corn; perhaps from being too close to Kerbal?!?!?
```
Comment (mwerle) [inline GameData/masTerTorch/Localization/en-us.cfg:102]: Copy-and-paste error or intentional to have the same text as SunInSpaceLow being part of this one?

File: README.md [hunk 9]
```diff
@@ -25,15 +26,17 @@ Lots of results for the new experiment.
 4. KDEX is part of the "Space Exploration" tech node.
 
 ## License
-This mod is licensed under the [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) license; effectively it means it's free to use and modify for non-commercial usage as long as the original author(s) are attributed.
-
+This mod is licensed under the [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) license; effectively it means it's free to use and modify for non-commercial usage as long as the original author(s) are attributed.
```
Comment (mwerle) [inline README.md:29]: Trailing whitespace ;) (/pedantic)

**Comment by mwerle** [comment]
Looks awesome, thanks @zer0Kerbal !  Haven't actually tried it yet (just had a quick look through the changes) but apart from a few minor issues it's looking great. Thank you for your time!

**Comment by mwerle** [comment]
> Cebu? Love diving - just haven't done it for a spell.

No, I was in Okinawa (Yonaguni to be precise) diving with hammerhead sharks :)

I was near Cebu last Golden Week diving with thresher sharks on Malapascua. That was pretty damn awesome!

**Comment by zer0Kerbal** [comment]
>  few minor issues

please, tell me so I can continually improve.

 I understand about the localization. My suggestion is release two versions this one time. One for pre 1.7.0 and one for 1.70+. Label the pre 1.7 as last version for it. Not like it will change much. Release it one after another and that way one is marked old and one is new - and CKAN doesn't get confused. hopefully am clearly communicating.

**Comment by zer0Kerbal** [comment]
I see that you did. thank you!

File: GameData/masTerTorch/Localization/en-us.cfg (hunk 8 above)
Comment (zer0Kerbal) [reply GameData/masTerTorch/Localization/en-us.cfg:102]: Am not very creative and sun in space would be -- well the same, only hotter. :D

File: Changelog.cfg (hunk 7 above)
Comment (zer0Kerbal) [reply Changelog.cfg:16]: in change logs, especially when in situations like this - I often over-communicate - the changelog was for you - am from a pre-git universe. 📟 so often best way was document changes verbatim. 

If too much, no worries. truncate as you see fit (obviously).

File: GameData/masTerTorch/Localization/en-us.cfg (hunk 2 above)
Comment (zer0Kerbal) [reply GameData/masTerTorch/Localization/en-us.cfg:11]: oversight. yes, case sensitive. kindly choose one, but as you probably know, change universally.

If being correct: probably KDEx would be correct. old keyboard probably messed up from old fingers.

File: GameData/masTerTorch/kdex.version (hunk 4 above)
Comment (zer0Kerbal) [reply GameData/masTerTorch/kdex.version:15]: agree. I just did a minor bump to communicate it to you; 👴  🏫

File: GameData/masTerTorch/Parts/kdex/part.cfg (hunk 3 above)
Comment (zer0Kerbal) [reply GameData/masTerTorch/Parts/kdex/part.cfg:24]: darn fingers... or was it PEBKAC.

File: README.md (hunk 5 above)
Comment (zer0Kerbal) [reply README.md:42]: am honored. thank you.
just a hidden comment - 👍 
just hide a "zed'K was here" somewhere - like in the sun. 📦

File: Changelog.cfg (hunk 6 above)
Comment (zer0Kerbal) [reply Changelog.cfg:10]: 😭 mistyping that was not cool fingers!

File: README.md (hunk 5 above)
Comment (zer0Kerbal) [reply README.md:42]: [zed'K was here](https://forum.kerbalspaceprogram.com/index.php?/profile/190933-zer0kerbal/)

**Comment by mwerle** [comment]
Hey @zer0Kerbal, so will you fix (some of) the issues I commented on or should I?  If you go ahead fix them, just push to your branch and they will be automatically added to the pull request.

File: GameData/masTerTorch/Localization/en-us.cfg (hunk 2 above)
Comment (mwerle) [reply GameData/masTerTorch/Localization/en-us.cfg:11]: Actually I'd prefer KDEX_ as the prefix.

File: GameData/masTerTorch/kdex.version (hunk 4 above)
Comment (mwerle) [reply GameData/masTerTorch/kdex.version:15]: Fair enough, thanks :)

File: Changelog.cfg (hunk 7 above)
Comment (mwerle) [reply Changelog.cfg:16]: Ah I see, and much appreciated.  I too hail from pre-git, but nowadays find modern diff tools to give me all the details I need with a human changelog reduced to pointing out any subtle changes and providing high-level summaries.  With git, best practice is to have each (set of) changes as a single commit. 

In practice I still tend to code away, and then try to unpick logically-cohesive commits afterwards - again, luckily this is one of git's strong points especially with a powerful gui like Git Extensions.

**Comment by zer0Kerbal** [comment]
*chuckle* shows the pre-git training. All good. 😃 It’s not like you have to go line-by-line and replace - *ahhh* the invent of the find-replace. I remember it well. 😝

**Comment by zer0Kerbal** [comment]
I should be able to get to it shortly – am digging through a couple other mods – pushing for the release of DuoPods/Yarbrough/RustyRockets/MoarKerbals/SimpleLife(++++) – I try to remain focused, but everytime I try, they pull me back away.. 😲 😃

**Comment by mwerle** [comment]
Wasn't sure when you'd get around to it so I've merged it and made the fixes myself.

Will probably move "Assets" back into the kdex part folder though. Unless there's a really good reason you split that up?  Generally speaking the assets for a part cannot be used anywhere else.

**Comment by zer0Kerbal** [comment]
Thank you – been so backed up (and continue to be)
I have a standard mod folder layout – and that is one of the things. Doesn’t matter either way; it’s your mod. 😃
